mvc/controllers/article.controller.js

```diff
@@ -24,3 +29,13 @@ exports.getArticleComments = (request, response, next) => {
     })
     .catch(next);
 };
+
+exports.postArticleComment = (request, response, next) => {
+  const articleId = request.params.article_id;
```

Comment:
`const id = req.params.article_id` can instead be destructured like 
`const {article_id} = req.params`